ssl/record/methods/dtls_meth.c

```diff
@@ -236,8 +238,9 @@ static int dtls_process_record(OSSL_RECORD_LAYER *rl, DTLS_BITMAP *bitmap)
     /* r->length is now just compressed */
     if (rl->compctx != NULL) {
         if (rr->length > SSL3_RT_MAX_COMPRESSED_LENGTH) {
-            RLAYERfatal(rl, SSL_AD_RECORD_OVERFLOW,
-                        SSL_R_COMPRESSED_LENGTH_TOO_LONG);
+            /* As per RFC 9147 4.5.2: Silently drop the packet in case of invalid length. */
```

Comment:
rfc 9147 is DTLS 1.3 which is not supported on master. I think it is okay to keep that refeerence, but could you add a reference for the DTLS 1.2 rfc: rfc6347 [4.1.2.7](https://datatracker.ietf.org/doc/html/rfc6347#section-4.1.2.7)
Thanks!